alws/utils/modularity.py

```diff
@@ -301,6 +301,10 @@ def iter_dependencies(self):
                 for stream in dep.get_buildtime_streams(module):
                     yield module, stream
 
+    def get_nsvca(self) -> str:
+        return (f'{self.name}:{self.stream}:{self.version}:'
+                f'{self.context}:{self.arch}')
```

Comment:
Please, make it property, just like the rest of attributes (name, stream, version, etc)